src/code_actions.zig

```diff
@@ -159,7 +188,7 @@ fn handleUnusedVariableOrConstant(builder: *Builder, actions: *std.ArrayListUnma
 
     try actions.append(builder.arena.allocator(), .{
         .title = "discard value",
-        .kind = .SourceFixAll,
+        .kind = .QuickFix,
```

Comment:
this should be keep at SourceFixAll

Reply:
I think your latest commit touched the wrong place. 
Set this location https://github.com/NuclearPhone/zls/blob/39ea5869a377eeec71fe333f2ee264fac7cf2bbb/src/code_actions.zig#L85 to `.QuickFix`.
And set this location https://github.com/NuclearPhone/zls/blob/39ea5869a377eeec71fe333f2ee264fac7cf2bbb/src/code_actions.zig#L176 to `.SourceFixAll`.